vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl

```diff
@@ -28,6 +28,13 @@
   owl:imports <http://qudt.org/2.1/vocab/dimensionvector> ;
   owl:versionIRI <http://qudt.org/2.1/vocab/quantitykind> ;
 .
+quantitykind:SpecificElectricCharge
+  rdf:type qudt:QuantityKind ;
+  qudt:hasDimensionVector <http://qudt.org/vocab/dimension/A0E1L0I0M-1H0T1D0> ;
```

Comment:
Apologies - I need to update our wiki documentation. Please replace with:

qudt:hasDimensionVector qkdv:A0E1L0I0M-1H0T1D0 ;